### PR TITLE
stdout/stderr logging option

### PR DESCRIPTION
**Pull request by mamico**
Using kubernetes we found sometimes useful to collect logging from pod using stdout/stderr instead of local files. 
This is a try to address this requirement.
If you think that could be useful and properly  implemented I could spread changes to the other docker images.

### CODE REVIEW

**Comment by jensens** [comment]
If this is a draft, you can open/convert the PR in/to draft mode, if this is ready for merge please remove draft statement from title. Anyway, a non-draft PR with uppercase draft in title is confusing.

**Comment by avoinea** [comment]
@mamico Niice! I think this should be the default behavior. 

Logging into files is not in accordance with [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)

But we didn't have this option when we first created this image: https://github.com/docker-library/official-images/pull/1434#issuecomment-220105913

File: 5.2/5.2.4/debian/docker-initialize.py [hunk 1]
```diff
@@ -224,6 +224,8 @@ def buildout(self):
         versions = self.env.get("PLONE_VERSIONS",
                    self.env.get("VERSIONS", "")).strip().split()
 
+        stdlogging = self.env.get("STD_LOGGING")
```
Comment (avoinea) [inline 5.2/5.2.4/debian/docker-initialize.py:227]: Should go to default behavior.

**Review by avoinea — CHANGES_REQUESTED** [review]

File: 5.2/5.2.4/debian/docker-initialize.py (hunk 1 above)
Comment (mamico) [reply 5.2/5.2.4/debian/docker-initialize.py]: Hi @avoinea, sorry for the delay. I changed the default with  stdout/stderr logging. So the command to switch to legacy behavior might be:
`docker run -p8080:8080 -e FILE_LOGGING=1 plone:5.2,4-debian
`
If you agree, I could spread the changes to the others 5.2.4 images (alpine, ...). 
I think I leave the other versions of Plone as they are.

Comment (avoinea) [reply 5.2/5.2.4/debian/docker-initialize.py]: Yes. Usually we don't rebuild existing tags, thus this will go into the next releases.

**Review by avoinea — APPROVED** [review]

**Pull request merged**